homeassistant/components/sensor/usps.py

```diff
@@ -107,6 +108,7 @@ def __init__(self, session, name):
         self._name = name
         self._attributes = None
         self._state = None
+        self.update()
```

Comment:
We don't want to [do IO in properties](https://github.com/home-assistant/home-assistant/issues/4210). What's the goal with these `update` calls? The `SCAN_INTERVAL` should provide sufficient updates, right?